scripts/add_gen3_repo.py

```diff
@@ -67,14 +70,21 @@ def main():
         gen2_repo = workspace.dataRepo
         gen2_calibs = workspace.calibRepo
         # Files stored in the Gen 2 part of the dataset, can be safely linked
+        # If curated calibs weren't handled with templates, convert them now
         _migrate_gen2_to_gen3(dataset, gen2_repo, gen2_calibs, gen3_repo, mode,
-                              curated=False,
+                              curated=(not hasTemplates),
                               config_file="convertRepo_calibs.py")
         # Our refcats and defects are temporary files, and must not be linked
         _migrate_gen2_to_gen3(dataset, gen2_repo, gen2_calibs, gen3_repo, mode="copy",
                               curated=False,
                               config_file="convertRepo_copied.py")
 
+    if not hasTemplates:
+        # ap_verify assumes a templates collection is present.
+        # Can't do this until gen3_repo exists.
+        butler = daf_butler.Butler(gen3_repo, writeable=True)
+        butler.registry.registerCollection("templates/deep", daf_butler.CollectionType.RUN)
```

Comment:
As discussed elsewhere, it may be better to change the hard-coded "templates/deep" here and below to something grabbed from a config file or other variable.

Reply:
Actually, it looks like my knowledge was out of date -- `ap_verify` tests run fine even if `ap_verify_testdata` no longer has any template collection. So I'll just remove the code.